source/includes/kubernetes/_k8_persistentvolumeclaims.md

```diff
@@ -236,3 +236,30 @@ Return value:
 | -------------------------- | ---------------------------------------------------------------- |
 | `taskId` <br/>_string_     | The id corresponding to the create persistent volume claim task. |
 | `taskStatus` <br/>_string_ | The status of the operation.                                     |
+
+<!-------------------- DELETE a persistent volume claim  -------------------->
+#### Delete a persistent volume claim
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/persistentvolumeclaims/cmc-staging-mysql/cmc-stg"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumeclaims/:id</code>
```

Comment:
should probably write that the `:id` at the end is required  // that its the id of the pvc to delete?

Reply:
We don't have a [Path parameters](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#delete-storageclass-v1-storage-k8s-io) section in our docs for K8 and other plugins.
- the `:service_code` and `:environment` are taken care of by the href
- we have a "Required" section but that's used only for fields required in a json body

So right now the `:id` is implied, should we create an improvement issue to add a "Path parameters" section or something similar?